src/qibolab/instruments/erasynth.py

```diff
@@ -11,204 +11,111 @@
 from qcodes_contrib_drivers.drivers.ERAInstruments import ERASynthPlusPlus
 from qibo.config import log
 
-from qibolab.instruments.abstract import InstrumentException
-from qibolab.instruments.oscillator import LocalOscillator, LocalOscillatorSettings
+from qibolab.instruments.oscillator import DummyDevice, LocalOscillator
 
-MAX_RECONNECTION_ATTEMPTS = 10
-TIMEOUT = 10
 
+class ERASynthEthernet(DummyDevice):
```

Comment:
Following the previous comment, this should inherit the qcodes abstract `Instrument`. Here I just wrote it in a way that is equivalent to what we currently have, but maybe qcodes already has a driver for the ethernet case, so we could drop this completely.

Reply:
Ok, I'd suggest raising an issue for this, and face it in a separate PR.

Making one step at a time is definitely a good idea, so if this is actually already reproducing the current Qibolab driver, so much the better.

Reply:
However, since you're overwriting all the methods, and conceptually is not a dummy device, you could drop the inheritance without anything changing in practice.